sentry_sdk/integrations/starlite.py

```diff
@@ -76,57 +78,52 @@ def patch_app_init() -> None:
     """
     old__init__ = Starlite.__init__
 
-    def injection_wrapper(self: "Starlite", *args: "Any", **kwargs: "Any") -> None:
-        after_exception = kwargs.pop("after_exception", [])
-        kwargs.update(
-            after_exception=[
-                exception_handler,
-                *(
-                    after_exception
-                    if isinstance(after_exception, list)
-                    else [after_exception]
-                ),
-            ]
-        )
+    @ensure_integration_enabled(StarliteIntegration, old__init__)
+    def injection_wrapper(self, *args, **kwargs):
+        # type: (Starlite, *Any, **Any) -> None
+        kwargs["after_exception"] = [
+            exception_handler,
+            *(kwargs.get("after_exception") or []),
+        ]
```

Comment:
Need to be careful here because even though in Litestar `after_exception` is always a sequence, it was not always that way in Starlite: https://github.com/litestar-org/litestar/blob/7b45ea74ac9dd808c751dc25a000199e56a61df5/starlite/config/app.py#L54 So we still need to support the case of the original `after_exception` being a single handler as opposed to a list of handlers.

Reply:
Good catch. I undid that particular change.